### ISSUE TITLE
Why the Wtv is so big?

### ISSUE THREAD

**Issue by chenghaoDong666** [Li-Chongyi/Zero-DCE_extension#9]
In the paper, it says that Wcol =0.5 and WtvA=20, while other loss's W=1. But in the code, WtvA is set = 1600, it is really strange...
![image](https://user-images.githubusercontent.com/38023034/145792577-b16bd76f-aece-4b47-b67e-4b77a15d4c8e.png)

**Comment by soumyadeep17** [comment]
Ya I also have the same doubt

**Comment by tkone2018** [comment]
@chenghaoDong666 @soumyadeep17 i also met the doubt, do you get it ?

**Comment by tkone2018** [comment]
the L_TV loss,return self.TVLoss_weight*2*(h_tv/count_h+w_tv/count_w)/batch_size,i wwant to know waht is 2?

**Comment by EthanChou860314** [comment]
`
      E = 0.6
      enhanced_image,A  = DCE_net(img_lowlight)
      Loss_TV = 1600*L_TV(A)
      # Loss_TV = 200*L_TV(A)			
      loss_spa = torch.mean(L_spa(enhanced_image, img_lowlight))
      loss_col = 5*torch.mean(L_color(enhanced_image))
      loss_exp = 10*torch.mean(L_exp(enhanced_image,E))
      # best_loss
      loss =  Loss_TV + loss_spa + loss_col + loss_exp
`

the code have # Loss_TV = 200*L_TV(A)  and  loss_col = 5*torch.mean(L_color(enhanced_image))
maybe 20 : 0.5 become 200 : 5   
???

does anyone saw the scale of loss_spa and loss_exp on paper?

**Comment by EthanChou860314** [comment]
@world2025 
do you have answer?
the L_TV loss,return self.TVLoss_weight2(h_tv/count_h+w_tv/count_w)/batch_size,i wwant to know waht is 2?
